tests/TestCase/Model/Behavior/SearchBehaviorTest.php

```diff
@@ -68,23 +68,28 @@ public function testProcessFilters()
         $behavior = $this
             ->getMockBuilder('Search\Model\Behavior\SearchBehavior')
             ->setConstructorArgs([$this->Comments])
-            ->setMethods(['_getAllFilters'])
+            ->setMethods(['_getAllFilters', '_flattenParams'])
```

Comment:
LOL, you mocked the method you made changes too allowing you to assert any input and output you want. Lines 170, 171 which are the main change you made are never really executed by the test suite.